Src/BootCamp.Chapter/LogMessage.cs

```diff
@@ -0,0 +1,37 @@
+﻿using BootCamp.Chapter.Logger;
+using System.IO;
+
+namespace BootCamp.Chapter
+{
+    public class LogMessage
+    {
+        private const string LogPath = @"..\..\..\Output\Log.txt";
+
+        public LogMessage()
+        {
+            File.Delete(LogPath);
+        }
+
+        private const int ConsoleLogger = 1;
+        private const int FileLogger = 2;
+        private int LogOption = FileLogger; //Set to either ConsoleLogger or FileLogger
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Make 'LogOption' 'readonly'.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=5147695)